src/Components/Backoffice/Backoffice.js

```diff
@@ -1,8 +1,37 @@
+import React from 'react';
```

Comment:
this file name should be router

Reply:
this router should be embebbed on the general router

Reply:
Fixed